plugins/faustwp/faustwp.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Plugin Name: FaustWP
+ * Plugin Name: Faust
```

Comment:
Just to be sure, modifying the plugin name should do no harm to the WP repo right?

Reply:
Correct. Updating "Plugin Name" for WordPress plugins will update the plugin title in WordPress,

><img width="813" alt="Screen Shot 2022-10-10 at 11 11 54 AM" src="https://user-images.githubusercontent.com/6676674/194898817-d60fd9ce-dd6e-4e0c-90d8-3feb675352a3.png">

as well as the plugin title on https://wordpress.org/plugins/faustwp.

><img width="1173" alt="Screen Shot 2022-10-10 at 11 12 22 AM" src="https://user-images.githubusercontent.com/6676674/194898828-c5db820e-27e5-464b-a029-7bc18d4b5844.png">